### DISCUSSION TITLE
windows可以源码运行吗

### DISCUSSION THREAD

**Discussion by agent-1900** [lllyasviel/Fooocus#3478] [Q&A] [answered]
如题。

**Comment by mashb1t** [comment, marked as answer]
Yes, but the zip file installation method is the easiest.
You can manually set up either a python venv, an anaconda env or install it locally by following the linux installation methods, except for calling the activate script directly you have to use the .exe files.
Please refer to documentation of the installation method in other repositories of the official sources.
